frontend/src/metabase/browse/tables/TableBrowser/TableBrowser.jsx

```diff
@@ -46,15 +42,17 @@ export const TableBrowser = ({
   xraysEnabled,
   showSchemaInHeader = true,
 }) => {
+  const databaseCrumb = useDatabaseCrumb(dbId);
+
   return (
     <>
       <BrowseHeaderContent>
         <BrowserCrumbs
           crumbs={[
             { title: t`Databases`, to: "/browse/databases" },
-            getDatabaseCrumbs(dbId),
+            databaseCrumb,
             showSchemaInHeader && { title: schemaName },
-          ]}
+          ].filter(Boolean)}
```

Comment:
When the `databaseCrumb` is loading or errors out, this will show a shorter breadcrumb trail.

<img width="214" alt="Screenshot 2024-12-10 at 18 07 14" src="https://github.com/user-attachments/assets/6fc468cd-798e-45d6-a03a-f31e8de13f5b">
<img width="330" alt="Screenshot 2024-12-10 at 18 07 20" src="https://github.com/user-attachments/assets/b26239f1-f8ec-4175-a40b-4bebad40f4d6">

I think we should show a placeholder instead to avoid confusion.

Reply:
> When the databaseCrumb is loading or errors out, this will show a shorter breadcrumb trail.

d29e5130ef15e290e34f023d27c868944251f042 + e3eff706d3cab17df75d48a6dbc2e0ea26c8de35

I used data from the store as a fallback - now error and loading state are not relevant anymore.
If initial global GET `/api/database` call fails, then app won't work anyway.

> I think we should show a placeholder instead to avoid confusion.

I fully agree but it's out of scope of this PR.